bibsonomy-webapp-v2/.env.development

```
@@ -0,0 +1,11 @@
+# Development environment configuration
+
+# API Configuration
+VITE_API_BASE_URL=http://localhost:8080
+
+# MSW Configuration - Enable mocks by default in development
+VITE_ENABLE_MOCKS=true
+
+# Application Configuration
+VITE_APP_TITLE=BibSonomy (Dev)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)